docs/plugin-development.rst

```diff
@@ -591,17 +610,26 @@ parent method. The prototypal nature of JavaScript makes that pattern impossible
 instead, we'll use a slightly less straightforward but equally powerful
 mechanism. This is best demonstrated by example. Let's say we want to execute
 some code any time the core ``HierarchyWidget`` is rendered, for instance to
-inject some additional elements into the view. We use the ``girder.wrap``
+inject some additional elements into the view. We use Girder's ``wrap`` utility
 function to `wrap` the method of the core prototype with our own function.
 
 .. code-block:: javascript
 
-    girder.wrap(girder.views.HierarchyWidget, 'render', function (render) {
+    import HierarchyWidget from 'girder/views/widgets/HierarchyWidget';
+    import { wrap } from 'girder/utilities/PluginUtils';
+
+    // Import your template file from your plugin
+    import myTemplate from 'girder_plugins/cats/templates/hierachyWidgetExtension.pug';
+
+    // CSS files pertaining to this view should be imported as a side-effect
+    import 'girder_plugins/cats/stylesheets/hierarchyWidgetExtension.styl';
```

Comment:
I would probably recommend using relative path here, I found it more portable. That's what I used when I ported all the plugins. The `girder_plugins/` path does work of course, but I feel people who are not familiar with Webpack aliases may wonder where this comes from. It would look less "magical" without it, e.g.:
```javascript
import myTemplate from '../templates/hierachyWidgetExtension.pug'; 
import '../stylesheets/hierarchyWidgetExtension.styl'; 
```
and maybe we can introduce the "recommended" tree structure for plugins at the same time, e.g., considering `plugins/cats/web_client`:
```
collections/
models/
stylesheets/
  hierarchyWidgetExtension.sty
templates/
  hierachyWidgetExtension.pug
views/
  hierachyWidgetExtension.js
main.js
routes.js
```